lib/utils/voip/callkeep_manager.dart

```diff
@@ -167,7 +168,26 @@ class CallKeepManager {
     Logs().v('[onPushKitToken] token => ${event.token}');
   }
 
-  Future<void> initialize() async {
+  Future<void> initialize({BuildContext? context}) async {
+    tryReadPhoneStatePermission().then((pStatus) {
+      if (context != null) {
+        if (pStatus == PermissionStatus.denied ||
+            pStatus == PermissionStatus.permanentlyDenied) {
+          ScaffoldMessenger.of(context).showSnackBar(
```

Comment:
What happens if a user will permanently deny the permission? Will he gets a snackbar on any app start with the same content?

Reply:
Okay, I'll add a different message to the snackbar for permanently denied.
What do you prefer more? After getting permanently denied, show the snack bar only once or everytime the user opens the app?